apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4PatientResourceProvider.java

```diff
@@ -269,6 +269,13 @@ public Bundle searchByCoverageContract(
     List<Beneficiary> matchingBeneficiaries =
         fetchBeneficiaries(coverageId, includeIdentifiersValues, paging);
 
+    // Insures backwards compatability
```

Comment:
Delete this comment as when we originally modified this file, we did not apply the +1 fix.  Now that the fix has been applied, this comment no longer applies.